### PR TITLE
Feat/add import screen

### PR DESCRIPTION
**Pull request by Ldoppea**
The old project sources have been imported in order to add ImportPage and in order to prepare next Install feature.

Webpack has been modified so jsx files are transpiled

### CODE REVIEW

File: src/cozy/wrappers/import-page/import-page.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ImportPage from "../../react/components/ImportPage";
+import ReactWrapper from "../react-wrapper";
+
+// wrap original ImportPage component into
+const ImportPageWrapper = ({ client, bitwardenData, ...props }) => {
+  return (
+    <ReactWrapper client={client} bitwardenData={bitwardenData} {...props}>
+      <ImportPage></ImportPage>
```
Comment (Crash--) [inline src/cozy/wrappers/import-page/import-page.jsx:9]: `<ImportPage />` :D

Comment (Crash--) [reply src/cozy/wrappers/import-page/import-page.jsx:9]: Does that mean that if we have to import another react component, we'll hav to wrap it inside a ReactWrapper too? It means that we'll mount several `ReactWrapper` at the same time, right?

File: src/scss/vault.scss [hunk 2]
```diff
@@ -1,4 +1,9 @@
-﻿@import "variables.scss";
+﻿/* Cozy UI utilities classes */
+@import '~cozy-ui/dist/cozy-ui.min.css';
+/* Cozy UI React components styles */
+@import '~cozy-ui/transpiled/react/stylesheet.css';
+
+@import "variables.scss";
```
Comment (Crash--) [inline src/scss/vault.scss:6]: Is it really related to `cozy-ui` styles?

**Review by Crash-- — APPROVED** [review]
CI is red but lint issues.

Looks good!

File: src/cozy/wrappers/import-page/import-page.jsx (hunk 1 above)
Comment (Ldoppea) [reply src/cozy/wrappers/import-page/import-page.jsx:9]: Yep that's it. It is working for now and I didn't notice any performance issue. But we may have to find a better way in the future if we are limited by this architecture.

File: src/scss/vault.scss (hunk 2 above)
Comment (Ldoppea) [reply src/scss/vault.scss:6]: I'm not sure, this comment is from the original code : https://github.com/cozy/cozy-passwords/blob/master/src/styles/index.css

Should I rewrite it with something like "Utility css classes" ?

Comment (Crash--) [reply src/scss/vault.scss:6]: My bad, I should have been clearer. My question was now about `cozy-ui` importation, but the latest import aka `variables.scss`. Why this import?

Comment (Ldoppea) [reply src/scss/vault.scss:6]: It's a false positive from the diff tool, this `@import` was already there.

**Pull request merged**